client/src/index.js

```diff
@@ -1,10 +1,17 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
+import { BrowserRouter } from 'react-router-dom';
+import ReactDOM from 'react-dom';   //Router Library
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+ReactDOM.render((
+  <BrowserRouter>   {/* Added Route to main website */}
```

Comment:
Can you convert this to History Mode instead of Hash Mode that we can properly handle different pages with a domain name.

Reply:
Yep, will do.

Reply:
From my research, I believe the App was already implemented in _History Mode_, when we invoke `BrowserRouter`. Implementing in _Hash Mode_ would be invoking `HashRouter`. If I'm wrong let me know.